Modified LogIn page, Home Page & created Reset Password page

Ade0211 opened the pull request:
1) Login Page done with a teacher login button which links to the login page when clicked
2) Homepage done with Login inputs which redirects to the next page when the username is validated

Ade0211 committed:
LogIn page & Home Page

LogIn page and Homepage done

Wolfjawan reviewed (approved):
LGTM 👍

Denis-Andrei commented on src/components/LoginComponent/LoginComponent.jsx:
```diff
@@ -1,5 +1,85 @@
+    fetch("https://stingy-cherry-sight.glitch.me/login", requestOptions)
+      .then((res) => res.json())
+      .then((res) => {
+        console.log(res);
```
I think the console.log can be removed

Ade0211 replied:
I have removed it

ISTANBULBEKLE replied:
Great Job Guys, you are HEROs

Denis-Andrei commented on src/components/LoginComponent/LoginComponent.jsx:
```diff
@@ -1,5 +1,85 @@
+        console.log(res);
+        if (res.success === true) {
+          localStorage.setItem("token", res.token); //storing the token in localStorage.
+          history.push("/student-tracker");
```
This route should go to /choose-class not /student-tracker

Ade0211 replied:
I only used it for a test but i have now changed it

Denis-Andrei commented on src/components/LoginComponent/LoginComponent.jsx:
```diff
@@ -1,5 +1,85 @@
+          </li>
+        </ul>
+      </form>
+      <footer className="page-footer">
```
Footer can be removed

Ade0211 replied:
The footer has been removed

Denis-Andrei commented on src/components/LoginComponent/logincomponent.scss:
```diff
@@ -0,0 +1,70 @@
+@import "../../assets/sass/main.scss";
+
+.login-box {
```
Maybe will be better if you center this component in the Login Page and not here

Ade0211 replied:
Thanks. done

Denis-Andrei commented on src/pages/Front/front.scss:
```diff
@@ -1,4 +1,39 @@
+    position: relative;
+    overflow: hidden;
+  }
+  &-button {
```
You can create a Button component and use it in both pages( front and login)

Ade0211 replied:
Thanks for that. I have created it and made new commits

Ade0211 committed:
test

Ade0211 committed:
Merge branch 'master' of https://github.com/CodeYourFuture/cyf-student-tracker into cyf-student-tracker-FrontEnd

Ade0211 committed:
Modified Front,Login Page& created reset password pageR

1) created a resusable button for both homepage and Login page
2) I removed the footer
3) I created the reset password page

ISTANBULBEKLE commented:
Guys, also thanks to putting many notes for us to follow

ISTANBULBEKLE reviewed (approved):
Great Job Guys. Go For Mars

Denis-Andrei reviewed (approved):
Looks good.
